ios/mapbox_maps_flutter/Sources/mapbox_maps_flutter/Classes/Extensions.swift

```diff
@@ -352,6 +375,53 @@ extension MbxEdgeInsets {
 
 // Mapbox to FLT
 
+extension MapboxMaps.PerformanceStatistics {
+    func toFLTPerformanceStatistics() -> PerformanceStatistics {
+        return .init(
+            collectionDurationMillis: collectionDurationMillis,
+            mapRenderDurationStatistics: mapRenderDurationStatistics.toFLTDurationStatistics(),
+            cumulativeStatistics: cumulativeStatistics?.toFLTCumulativeRenderingStatistics(),
+            perFrameStatistics: perFrameStatistics?.toFLTPerFrameRenderingStatistics()
+        )
+    }
+}
+
+extension MapboxMaps.DurationStatistics {
+    func toFLTDurationStatistics() -> DurationStatistics {
+        return .init(maxMillis: maxMillis, medianMillis: medianMillis)
+    }
+}
+
+extension MapboxMaps.CumulativeRenderingStatistics {
+    func toFLTCumulativeRenderingStatistics() -> CumulativeRenderingStatistics {
+        return .init(
+            drawCalls: drawCalls.map(Int64.init),
+            textureBytes: textureBytes.map(Int64.init),
+            vertexBytes: vertexBytes.map(Int64.init),
+            graphicsPrograms: __graphicsPrograms?.int64Value,
+            graphicsProgramsCreationTimeMillis: __graphicsProgramsCreationTimeMillis?.doubleValue,
+            fboSwitchCount: __fboSwitchCount?.int64Value
+        )
+    }
+}
+
+extension MapboxMaps.PerFrameRenderingStatistics {
+    func toFLTPerFrameRenderingStatistics() -> PerFrameRenderingStatistics {
+        return .init(
+            topRenderGroups: topRenderGroups.map(\.fLTGroupPerformanceStatistics),
+            topRenderLayers: topRenderLayers.map(\.fLTGroupPerformanceStatistics),
+            shadowMapDurationStatistics: shadowMapDurationStatistics.toFLTDurationStatistics(),
+            uploadDurationStatistics: uploadDurationStatistics.toFLTDurationStatistics()
+        )
+    }
+}
+
+extension MapboxMaps.GroupPerformanceStatistics {
+    var fLTGroupPerformanceStatistics: GroupPerformanceStatistics {
```

Comment:
should we match the convention of converting from MapboxMaps to FLT as in the other conversion here?
```suggestion
    func toFLTGroupPerformanceStatistics() -> GroupPerformanceStatistics {
```